docs/using_packet.md

```diff
@@ -181,27 +182,38 @@ Invoking Syntex From Your Build Script
 --------------------------------------
 
 To pull everything together, we need to invoke `syntex` within the build script.
-This is a simple build script for building just a single packet type; see the
-`build.rs` from `libpnet` for an example of how to build a bunch of them:
+This is a simple build script for recursively building files using the pnet_macros; see also the
+`build.rs` from `libpnet` as an example of how many packets are built:
```

Comment:
I'm not sure this comment makes sense - this example previously only built a single file, it then referred the reader to the `build.rs` script for building multiple files.

I would either keep this example only using a single file, and leave the original text; remove the comment about `build.rs`; or change the comment to specify it as a more complete example - though it looks copy/pasted so maybe not?